### DISCUSSION TITLE
spans

### DISCUSSION THREAD

**Discussion by Corvette653** [f35f22fan/Ods2#4] [General]
Hi I have been using your library for few days and I have problem with 
cell->number_rows_spanned(int rows);
it works if the cell has spanned both columns and rows, but if I want to span only rows it doesn't

code example:
```
    auto *book = ods::Book::New();
    ods::AutoDelete<ods::Book*> ad(book);
    auto *spreadsheet = book->spreadsheet();
    auto *sheet = spreadsheet->NewSheet("Sheet name"); // creating new sheet
    auto *row = sheet->NewRowAt(0);
    auto *cell = row->NewCellAt(0);

    cell->SetFirstString("first cell");
    cell->number_rows_spanned(3);                                    // code which doesn't work

    QFile file("c:\\");
    QString err;                                                                      // saving file
    book->Save(file, &err);
```

**Comment by f35f22fan** [comment]
Hi, I assume you're using MS Office because here in LibreOffice on Linux your example works fine. What version of MS Office you're using? Asking in case I reboot to windows and it works fine too.

**Reply by Corvette653** [reply]
I use Windows 10 and MS Office 2019

**Comment by f35f22fan** [comment]
Committed a fix to master, should be working fine now. Please let me know if it doesn't work for you.

**Reply by Corvette653** [reply]
It works fine :)
Thank you for this library, it's really useful

**Reply by f35f22fan** [reply]
I'm glad it works, I was wondering how do you find it building this project on windows? Did you spend much time figuring it out? Because when I rebooted to windows I had quite a struggle, first it turned out my VS Studio trial expired and so I had to try to build it for mingw, then it turned out some of the functions from ods/eval.hh didn't have the ODS_API markers to export them to the library (which I fixed in the same commit) and so the compiler told me it can't find those functions, I was wondering if 9/10 people trying out this project just give up half way.

**Reply by Corvette653** [reply]
I spent much time to build this(I use qt) but it's mainly my fault.
I am beginner, writing code is ok, but if I have to do something around it like installing additional libraries or just configure environment for the first time, I am terrified.
Your instruction is very useful, I would give up without it.

**Reply by f35f22fan** [reply]
Ok, thanks.
